fullscreen.bs

```diff
@@ -314,8 +314,8 @@ attribute's getter must run these steps:
 <ol>
  <li><p>Let <var>promise</var> be a new promise.
 
- <li><p>If <var>doc</var>'s <a>fullscreen element</a> is null, reject <var>promise</var> with a
- <code>TypeError</code> exception, and return <var>promise</var>.
+ <li><p>If <var>doc</var>'s <a>fullscreen element</a> is null, fulfill <var>promise</var> with
+ undefined, and return <var>promise</var>.
```

Comment:
Ugh, I see I used "fulfill" in a couple places instead of "resolve". Do you want me to fix those first in a PR?